src/inject/dynamic-theme/image.ts

```diff
@@ -78,6 +78,23 @@ function analyzeImage(image: HTMLImageElement) {
         logWarn(`logWarn(Image is empty ${image.currentSrc})`);
         return null;
     }
+
+    // Get good appromized image size in memory terms.
+    // Width * Height * 4(R, G, B, A) and 1.33 because rgba can contain up to 3 digits.
+    const size = naturalWidth * naturalHeight * 4 * 1.33;
```

Comment:
1.33 should take into account for the browser's memory on having an compressed version and some other metadata which contributes to the memory of such image and some other values I don't remember.

Reply:
OK, it can have metadata, but I think it should not correlate with width and height. I think it should be some static length, e.g. 500B, so you could just skip it.